build.js

```diff
@@ -71,6 +72,61 @@ const parseTypography = (obj = {}) => {
   };
 };
 
+const formatTwTypographyValues = (obj) => {
```

Comment:
Can we keep the names of these functions the same? i.e. `flattenTwTypographyValues` ?

Reply:
Oops, good catch!